cmd/kubectl-gadget/advisor/seccomp.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 var seccompAdvisorCmd = &cobra.Command{
```

Comment:
nit: I think it can be renamed to just `seccompPolicyCmd` as well.

Reply:
This was on purpose, I did not rename any variable, for example there is still `execSnoopCmd` in `exec.go`.
This is let as future work.